# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_proxyconnect.js
# Commit: b45bd4ea0000
# Full Hash: b45bd4ea0000e64e4b159cac01d09519d73166f7
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-17 09:39:01
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r 705439f1d269 -r b45bd4ea0000 netwerk/test/unit/test_proxyconnect.js
--- a/netwerk/test/unit/test_proxyconnect.js	Wed Nov 16 16:37:29 2022 +0000
+++ b/netwerk/test/unit/test_proxyconnect.js	Wed Nov 16 16:37:29 2022 +0000
@@ -255,7 +255,9 @@
     streamIn = transport
       .openInputStream(0, SEGMENT_SIZE, SEGMENT_COUNT)
       .QueryInterface(Ci.nsIAsyncInputStream);
-    streamOut = transport.openOutputStream(0, 0, 0);
+    streamOut = transport
+      .openOutputStream(0, 0, 0)
+      .QueryInterface(Ci.nsIAsyncOutputStream);
 
     streamIn.asyncWait(connectHandler, 0, 0, threadManager.mainThread);
   } catch (e) {